src/v/cluster/controller.h

```diff
@@ -56,6 +56,13 @@ class controller {
         return _security_frontend;
     }
 
+    ss::sharded<data_policy_frontend>& get_data_policy_frontend() {
+        return _data_policy_frontend;
+    }
+    data_policy_manager& get_data_policy_manager() {
```

Comment:
drop the `get_` prefix. nothing wrong with it, but it matches the style of the rest of the code base